src/components/Look1/Look1.vue

```diff
@@ -69,81 +89,81 @@
 </template>
 
 <script>
-import Vue from "vue";
-import VueResource from "vue-resource";
+import Vue from 'vue'
+import VueResource from 'vue-resource'
 
-Vue.use(VueResource);
+Vue.use(VueResource)
 
-function fakeItems() {
+function fakeItems () {
   return [
     {
       id: 1,
-      name: "Voluneteer 1",
+      name: 'Voluneteer 1',
       children: [
         {
           id: 1.1,
-          name: "Volunteer 1.1"
+          name: 'Volunteer 1.1'
         },
         {
           id: 1.2,
-          name: "Volunteer 1.2"
+          name: 'Volunteer 1.2'
         }
       ]
     },
     {
       id: 2,
-      name: "Voluneteer 2"
+      name: 'Voluneteer 2'
     },
     {
       id: 3,
-      name: "Voluneteer 3"
+      name: 'Voluneteer 3'
     }
-  ];
+  ]
 }
 
 export default {
   data: () => ({
     items: [],
     tree: [],
     form: {
-      name: "",
-      email: "",
-      notes: "",
-      selected: ""
+      name: '',
+      email: '',
+      notes: '',
+      selected: ''
     }
   }),
-  methods: {
-    cansubmit: function() {
-      return (
-        this.form.name !== "" &&
-        this.form.email !== "" &&
-        this.tree !== null &&
-        this.tree.length > 0
-      );
-    },
-    submit: function() {
-      alert(this.form.name + " Applied to volunteer " + this.tree);
-    },
-    updatetree: function() {
-      alert(this.active);
+  computed: {
+    filter () {
+      return this.caseSensitive
+        ? (item, search, textKey) => item[textKey].indexOf(search) > -1
+        : undefined
     }
   },
-  mounted() {
-    this.$http.get("http://localhost:3000").then(
+  mounted () {
+    this.$http.get('http://localhost:3000').then(
       () => {
-        this.items = fakeItems();
+        this.items = fakeItems()
       },
       () => {
-        this.items = fakeItems();
+        this.items = fakeItems()
       }
-    );
+    )
   },
-  computed: {
-    filter() {
-      return this.caseSensitive
-        ? (item, search, textKey) => item[textKey].indexOf(search) > -1
-        : undefined;
+  methods: {
+    cansubmit: function () {
```

Comment:
Do you mind switching these to camelCase?

Reply:
Linting in this project is complicated because of vue, which adds its own parser and linting settings, so that might be part of it